.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8u171-jdk-node-browsers
```

Comment:
nit: would be feasible to use a Yaml pointer (I think they are called) (https://coderwall.com/p/wqgm7g/using-pointers-to-merge-elements-in-yaml-file) to reuse these configs? Not sure if it's possible to merge pointers between them for the particular cases.

Reply:
I know lots of people use yaml pointers now, but I think they're actually net negative for this use-case.  They introduce a bit more complexity (which nobody can write correctly off the top of their head, and nobody is 100% sure what they do just by eyeballing) just to eliminate one or two lines.  In practise, I think the duplication and super easy to read YAML is preferrable.

In order to make CircleCI config shorter, I've just used a denser syntax: 

```yaml
  compile:
    docker: [{ image: 'circleci/openjdk:...' }]
    resource_class: medium+
    steps:
```